client/src/templates/Challenges/classic/desktop-layout.tsx

```diff
@@ -171,7 +171,8 @@ const DesktopLayout = (props: DesktopLayoutProps): JSX.Element => {
   const projectBasedChallenge = hasEditableBoundaries;
   const isMultifileCertProject =
```

Comment:
```suggestion
  const isMultifileProject =
```
Bit of a nitpick, but since labs are not certification projects, I think this should reflect that.